src/main/java/com/kodilla/ecommercee/repository/ProductRepository.java

```diff
@@ -10,6 +10,15 @@
 @Repository
 @Transactional
 public interface ProductRepository extends CrudRepository<Product, Long> {
+
+
     List<Product> findByProductName(String name);
+
     List<Product> findByProductId(Long id);
```

Comment:
To powinno zwracać albo Optional<Product> albo Product, bo inaczej ta lista będzie zawsze jednoelementowa a to bez sensu a do tego miesza się w testach

Reply:
OK, dzięki!